controller/controller_server/utils.py

```diff
@@ -201,6 +236,34 @@ def generate_csi_create_snapshot_response(new_snapshot, source_volume_id):
     return res
 
 
+def _get_supported_capabilities(volume_capabilities):
+    if volume_capabilities.HasField(config.VOLUME_CAPABILITIES_FIELD_ACCESS_TYPE_MOUNT):
+        return csi_pb2.VolumeCapability(
+            mount=csi_pb2.VolumeCapability.MountVolume(fs_type=volume_capabilities.mount.fs_type),
+            access_mode=csi_pb2.VolumeCapability.AccessMode(mode=volume_capabilities.access_mode.mode))
+
+    elif volume_capabilities.HasField(config.VOLUME_CAPABILITIES_FIELD_ACCESS_TYPE_BLOCK):
+        return csi_pb2.VolumeCapability(
+            mount=csi_pb2.VolumeCapability.BlockVolume(),
+            access_mode=csi_pb2.VolumeCapability.AccessMode(mode=volume_capabilities.access_mode.mode))
+
+
+def generate_csi_validate_volume_capabilities_response(volume_context, volume_capabilities, parameters):
+    logger.debug("creating validate volume capabilities response")
+
+    capabilities = []
+    for capability in volume_capabilities:
+        capabilities.append(_get_supported_capabilities(volume_capabilities=capability))
```

Comment:
please avoid calling more than 1 function in the same line

Reply:
done